src/main/java/github/grace000/erversay/RequestRouter.java

```diff
@@ -1,48 +1,70 @@
 package github.grace000.erversay;
 
-import static github.grace000.erversay.Constants.*;
+import static github.grace000.erversay.Constants.Headers.*;
+import static github.grace000.erversay.Constants.Body.*;
+import static github.grace000.erversay.Constants.Paths.*;
+import static github.grace000.erversay.Constants.StatusCodes.*;
 
 public class RequestRouter {
-    private String code;
     private String status;
     private String body;
     private int contentLength;
+    private String headers;
 
     public Response route(Request request) {
         String path = request.path;
+        String method = request.method;
 
-        return getResponse(path);
+        return getResponse(path, method);
     }
 
-    private Response getResponse(String path) {
-        switch(path) {
-            case SIMPLE_GET_URI:
-                getResponseForSimpleGet();
-                break;
-            case PIGGLY_URI:
-                getResponseForPigglyGet();
-                break;
-            default:
-                getResponseForNotFound();
+    private Response getResponse(String path, String method) {
+        if (path.equals(SIMPLE_GET_URI) && method.equals("GET")) {
+            getResponseForSimpleGet();
+        } else if(path.equals(PIGGLY_URI) && method.equals("GET")) {
+            getResponseForPigglyGet();
+        } else if(path.equals(METHODS_ONE_URI)) {
+            if(isStandardOptionsRequest(method)) {
+                getResponseForOptions();
+            }
+        } else if(path.equals(METHODS_TWO_URI)) {
+            if(isStandardOptionsRequest(method) || method.equals("PUT") || method.equals("POST")) {
+                getResponseForOptionsTwo();
+            }
+        } else {
+            getResponseForNotFound();
         }
```

Comment:
This `if` conditional is getting pretty gnarly. What's your plan to clean it up?

Reply:
This is where I was going to use a switch statement based on the methods instead, and then cycle through and match up the incoming path.

Reply:
You gonna do it as part of this PR?

Reply:
No, I'd prefer to do it as part of the POST PR.

Reply:
Why?

Reply:
I should be on the post request this week. I can either add a card to refactor that portion of the options request or address it within the post request PR. I'd like to maintain the current sprint schedule.